Dockerfile

```diff
@@ -36,6 +38,8 @@ RUN sbt compile
 COPY src/main src/main
 COPY src/test src/test
 RUN sbt compile
+RUN sbt test
```

Comment:
Do we need to run regular tests on build? I'm not opposed to this in principle, but our normal PR process should catch any `sbt test` failures.